executor/aggregate_test.go

```diff
@@ -357,8 +357,8 @@ func (s *testSuite) TestGroupConcatAggr(c *C) {
 	result = tk.MustQuery("select id, group_concat(name SEPARATOR ',') from test group by id")
 	result.Check(testkit.Rows("1 10,20,30", "2 20", "3 200,500"))
 
-	result = tk.MustQuery("select id, group_concat(name SEPARATOR '%') from test group by id")
-	result.Check(testkit.Rows("1 10%20%30", "2 20", "3 200%500"))
+	result = tk.MustQuery(`select id, group_concat(name SEPARATOR '%') from test group by id`)
```

Comment:
maybe we should change all the " to \` in other `MustQuery` function calls ?

Reply:
We can do that step by step.

Reply:
OK

Reply:
Maybe I should amend it so that when we change this issue, we also change the other items of this file?

e.g `MustExec`,`Exec`, `MustQuery`